### PR TITLE
Standardize the code format

### PR DESCRIPTION
**Pull request by iamxiaojianzheng**
Avoid affecting pr due to different local formatting configurations.

### CODE REVIEW

**Review by gorillamoe — APPROVED** [review]

File: lua/kulala/globals/init.lua [hunk 1]
```diff
@@ -10,5 +10,6 @@ M.SCRATCHPAD_ID = "kulala://scratchpad"
 M.HEADERS_FILE = plugin_tmp_dir .. "/headers.txt"
 M.BODY_FILE = plugin_tmp_dir .. "/body.txt"
 M.COOKIES_JAR_FILE = plugin_tmp_dir .. "/cookies.txt"
+M.CONSOLE_FILE = plugin_tmp_dir .. "/console.txt"
```
Comment (gorillamoe) [inline lua/kulala/globals/init.lua:13]: This shouldn't be part of this PR, eh?

File: lua/kulala/ui/init.lua [hunk 2]
```diff
@@ -278,6 +278,24 @@ M.show_headers_body = function()
   end
 end
 
+M.show_console = function()
```
Comment (gorillamoe) [inline lua/kulala/ui/init.lua:281]: This is all from a different PR I guess.

**Review by gorillamoe — CHANGES_REQUESTED** [review]

File: lua/kulala/ui/init.lua (hunk 2 above)
Comment (iamxiaojianzheng) [reply lua/kulala/ui/init.lua]: Sorry, I revised the commit. The pr contains only code formatting.

File: lua/kulala/globals/init.lua (hunk 1 above)
Comment (iamxiaojianzheng) [reply lua/kulala/globals/init.lua]: Sorry, this is part of another PR. https://github.com/mistweaverco/kulala.nvim/pull/189

**Review by gorillamoe — APPROVED** [review]
